### PR TITLE
Css sass conversion

### PR DESCRIPTION
**Pull request by guswadsworth12**
Convert to Sass for updating!

Converted css to partials.

### CODE REVIEW

**Comment by phildominguez** [comment]
image
image

The sidebar seems to have been affected

**Comment by phildominguez** [comment]
I think `sass/actors/_car.scss` should be `sass/actors/_vehicles.scss`

File: sass/actors/_index.scss [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+@forward '../actors/stat-containers';
+
+@forward '../actors/tabs';
+
+@forward '../actors/zord';
+
+
+
+
```
Comment (phildominguez) [inline sass/actors/_index.scss:31]: Extra lines here

File: sass/assets/_index.scss [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+@forward '../assets/tox';
+
+@forward '../assets/variables';
+
+
+
+
+
+
```
Comment (phildominguez) [inline sass/assets/_index.scss:37]: Extra lines

Comment (guswadsworth12) [reply sass/assets/_index.scss]: removed extra lines

**Review by phildominguez — APPROVED** [review]
Looks great!

**Pull request merged**